### ISSUE TITLE
Attendance Badges are not working for groups in sub-areas

### ISSUE THREAD

**Issue by JimMichael** [SparkDevNetwork/Rock#2032]
### Prerequisites
* [x] Put an X between the brackets on this line if you have done all of the following:
    * Can you reproduce the problem on a fresh install or the [demo site](http://rock.rocksolidchurchdemo.com/)?
    * Did you include your Rock version number and [client culture](https://github.com/SparkDevNetwork/Rock/wiki/Environment-and-Diagnostics-Information) setting?
    * Did you [perform a cursory search](https://github.com/issues?q=is%3Aissue+user%3ASparkDevNetwork+-repo%3ASparkDevNetwork%2FSlack) to see if your bug or enhancement is already reported?

### Description

When you have check-in groups within an Area that is a Sub-Area, the Family Attendance and Family 16 Week badges do not reflect attendance in these groups. See https://github.com/SparkDevNetwork/Rock/issues/1835 for screen shots of a area/group hierarchy that highlights this issue.

### Steps to Reproduce

1. Check someone into a group that is in a sub-area of the Weekly Service area.
2. Observe that the attendance badges do not reflect this attendance.

**Expected behavior:** 

Attendance in groups that are in a sub-area would be counted in the Family Attendance badges

**Actual behavior:** 

Attendance in groups that are in a sub-area are ignored by the Family Attendance badges

### Versions

* **Rock Version:** 6.2 and all prior.
* **Client Culture Setting** en-US

**Comment by JimMichael** [comment]
Ok, I'm not sure if something changed or I missed this the first time around (and submitted this issue)... but the attendance badges (for groups in sub areas) are working now! To be honest, I had forgotten I created this ticket, and the other day I noticed the attendance badges weren't working for kids I knew had attended, so (apparently unlike before) I went to the group type of one of my areas and sure enough, _Weekend Service_ under the Attendance section of the Group Type was un-checked. Once I set that on all of the kids group types, the badges lit up! So maybe the bug is that when _creating_ a check-in structure with sub-areas, _Weekly Attendance_ is NOT enabled like it would be on a normal (non sub) area? In any case, it's working here now.
